app/views/RoomView/index.js

```diff
@@ -417,23 +419,27 @@ class RoomView extends React.Component {
 		if (editing || replying || reacting) {
 			return;
 		}
-		this.setState({ selectedMessage: {}, showActions: false });
+		this.setState({
+			replyMessage: {}, editMessage: {}, selectedMessage: {}, showActions: false
+		});
 	}
 
 	onErrorActionsHide = () => {
 		this.setState({ selectedMessage: {}, showErrorActions: false });
 	}
 
 	onEditInit = (message) => {
-		this.setState({ selectedMessage: message, editing: true, showActions: false });
+		this.setState({
+			editMessage: message, selectedMessage: {}, editing: true, showActions: false
```

Comment:
Here, I thought after selecting edit or reply option, I should make the `selectedMessage` as empty object for selecting new messages, but I can change it if needed.